### PR TITLE
Insticator: Update impression ext for prebid-server params 

### PR DESCRIPTION
**Pull request by shubhamc-ins**

### CODE REVIEW

File: modules/insticatorBidAdapter.js [hunk 1]
```diff
@@ -17,17 +17,17 @@ export const OPTIONAL_VIDEO_PARAMS = {
...
-  'skipafter': (value) => isInteger(value),
-  'sequence': (value) => isInteger(value),
+  'linearity': (value) => isInteger(value) && [1].includes(value),
   'battr': (value) => isArrayOfNums(value),
+  'skipafter': (value) => isInteger(value),
+  'skip': (value) => isInteger(value) && [1, 0].includes(value),
   'maxextended': (value) => isInteger(value),
-  'minbitrate': (value) => isInteger(value),
   'maxbitrate': (value) => isInteger(value),
-  'playbackmethod': (value) => isArrayOfNums(value),
```
Comment (patmmccann) [inline modules/insticatorBidAdapter.js:29]: Hi @shubhamc-ins this commit is poor conduct and will not be tolerated

Please resubmit

**Review by patmmccann — CHANGES_REQUESTED** [review]
Nonsense commits to hide from the duplication detector are poor conduct

**Comment by patmmccann** [comment]
Also take a look at #11970 which is working on a real solution to the duplicated code

File: modules/insticatorBidAdapter.js (hunk 1 above)
Comment (shubhamc-ins) [reply modules/insticatorBidAdapter.js:29]: @patmmccann apologies, I will resubmit.

**Pull request closed without merging**